recipes/dotnet_agent.rb

```diff
@@ -0,0 +1,86 @@
+agent = node['appdynamics']['dotnet_agent']
+controller = node['appdynamics']['controller']
+proxy = node['appdynamics']['http_proxy'] 
+bitness = node['kernel']['os_info']['os_architecture']
+install_directory = agent['install_dir']
+system_direcotry = node['kernel']['os_info']['windows_directory']
```

Comment:
Typo. You want `system_directory`. The recipe works because it's misspelled everywhere.